decidim-participatory_processes/app/cells/decidim/participatory_process_groups/content_blocks/title/show.erb

```diff
@@ -4,7 +4,7 @@
     <div class="columns medium-9">
       <%= decidim_sanitize_editor translated_attribute(participatory_process_group.description) %>
       <div class="row">
-        <div class="column medium-2 text-muted">
+        <div class="column medium-3 text-muted">
```

Comment:
I tested it and for me it still breaks when I keep narrowing the screen.

Maybe just 50/50 split or break them into their own rows? I think it would be fine even when broken into separate rows as it is on mobile.

Or you could do something with the flex helpers (in which case it does not break into its own row on mobile):

```html
<div class="flex--fsc">
  <div class="mr-sm text-muted">First</div>
  <div class="mr-sm text-muted">Second</div>
</div>
```

Reply:
You're right, it breaks again at some breakpoints.

For my part and own project, I will fix it with a "medium-5" that seems OK at all viewport sizes.